src/components/Book.js

```diff
@@ -26,14 +26,26 @@ const Book = (props) => {
         <button type="submit">Edit</button>
       </div>
       <div className="completion">
-        <p>
-          {book.percentageCompleted}
-          % Completed
+        <svg
+          className="percentageChart"
+          style={{
+            strokeDasharray: `${Math.floor(Math.random() * (150 - 100) + 100)} 999`,
+          }}
+          viewbox="0 0 100 100"
+          width="80"
+          height="80"
+          data-percent="100"
+        >
```

Comment:
- [ ] I suggest you try as much as possible to ensure that no inline styling is done in your project as this is considered bad practice.